cognee/infrastructure/databases/relational/create_relational_engine.py

```
@@ -39,8 +39,16 @@ def create_relational_engine(
-        )
+        try:
+            # Test if asyncpg is available
+            import asyncpg
+
+            connection_string = (
+                f"postgresql+asyncpg://{db_username}:{db_password}@{db_host}:{db_port}/{db_name}"
+            )
+        except ImportError:
+            raise ImportError(
+                "PostgreSQL dependencies are not installed. Please install with 'pip install cognee\"[postgres]\"' or 'pip install cognee\"[postgres-binary]\"' to use PostgreSQL functionality."
+            )
```

Ruff (0.13.1)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)